src/wp-content/themes/twentytwenty/inc/template-tags.php

```diff
@@ -659,7 +659,7 @@ function twentytwenty_nav_menu_social_icons( $item_output, $item, $depth, $args
 function twentytwenty_no_js_class() {
 
 	?>
-	<script>document.documentElement.className = document.documentElement.className.replace( 'no-js', 'js' );</script>
+	<script>document.documentElement.classList.remove("no-js");</script>
```

Comment:
This is just modernization change.

Reply:
Why is that necessary? It changes the behavior as far as I can tell, the `.js` selector wouldn't work anymore. Some themes (including potentially this one?) have certain JS or CSS that targets such a selector, so I don't think removing it is worth the risk.

Reply:
The `.js` selector is necessary for Twenty Twenty.

https://github.com/WordPress/wordpress-develop/blob/1118e37f48ff184c40bc4e593ae13b1fbe4b8001/src/wp-content/themes/twentytwenty/style.css#L1056-L1062

With other themes that replace `no-js` with `js`, even if the theme itself does not use the class, someone might use it in a child theme (as I have done with Twenty Fifteen and Twenty Sixteen).

Reply:
Oh yes, I neglected to add the `js`. Nevertheless, I'll just revert this change.